### DISCUSSION TITLE
How to group MDX docs in storybook?

### DISCUSSION THREAD

**Discussion by shellwe** [storybookjs/storybook#26801] [Help]
### Summary

I have a bunch of docs and I know how to group stories together, either with using showRoots: false in the manager.ts file or having "groupName/fileName" as the title, but neither of those seem to work with the hand coded MDX docs.

I went through the following two pages but didn't see any info how to organize it.
https://storybook.js.org/blog/structuring-your-storybook/
https://storybook.js.org/docs/writing-docs/doc-blocks

Is there something I'm missing to be able to do this?

**Comment by morriq** [comment]
You can use `Meta` component

`<Meta title="docs/README" />`
